addons/website_slides/views/website_slides_templates_homepage.xml

```diff
@@ -0,0 +1,429 @@
+            <span class="font-weight-bold" t-esc="achievement.user_id.name"/> achieved <a href="/slides"><span t-esc="achievement.badge_id.name"/></a>
+        </div>
+    </div>
+</template>
+
+<template id='slides_home_users_small' name="Latest Users">
+    <div class="o_wslides_home_aside">
+        <div class="row mt16 o_wslides_home_aside_title">
+            <div class="col">
+                <a href="slides" class="text-muted float-right">
+                    <small class="float-right">View all</small>
+                </a>
+                <h3>Heroes of the week</h3>
+                <hr/>
+            </div>
+        </div>
+        <div class="row">
+            <t t-if="users">
+                <div class="col">
+                    <t t-set="counter" t-value="1"/>
+                    <t t-foreach="users" t-as="user">
+                        <t t-call="website_slides.user_quickkarma_card"/>
+                        <t t-set="counter" t-value="counter + 1"/>
+                    </t>
+                </div>
+            </t>
+            <t t-else=""><p class="col mt8">No heroes currently :(</p></t>
+        </div>
+    </div>
+</template>
+
+<template id='user_quickkarma_card' name="User QuickKarma Card">
+    <div class="row no-gutters mt8 align-items-center">
+        <div class="col-2">
+            <span class="font-weight-bold text-muted o_wslides_home_circled" t-esc="counter"/>
+        </div>
+        <div class="col-2">
+            <t t-call="website_slides.slides_misc_user_image">
+                <t t-set="record" t-value="channels_popular[:1]"/>
+            </t>
+        </div>
+        <div class="col">
+            <span class="font-weight-bold" t-esc="user.name"/>
+            <div class="d-flex align-items-center">
+                <span class="text-muted" t-esc="user.rank_id.name"/><span class="ml8 mr8">&#8226;</span><span t-esc="user.karma" class="badge badge-success"/>
+            </div>
+        </div>
+    </div>
+</template>
+
+<template id='slides_home_user_profile_small' name="User Profile">
+    <div class="o_wslides_home_aside">
+        <div class="row mt16 o_wslides_home_aside_title align-items-center">
+            <div class="col">
+                <t t-call="website_slides.slides_misc_user_image">
+                    <t t-set="record" t-value="channels_popular[:1]"/>
+                </t>
+                <a t-att-href="'/slides'" class="float-right">Edit</a>
+                <h4 t-esc="user.name" class="font-weight-bold"/>
+                <hr/>
+            </div>
+        </div>
+        <div class="row mt8">
+            <p class="col">INSERT DBE TEMPLATE HERE</p>
+        </div>
+    </div>
+</template>
+
+<template id='slides_home_user_achievements_small' name="User Achievements">
+    <div class="o_wslides_home_aside mt16">
+        <t t-foreach="challenges" t-as="challenge">
+            <t t-set="challenge_done" t-value="challenge in challenges_done if challenges_done else False"/>
```

Comment:
t-value looks weird but my brain doesn't compute anymore.

Reply:
Too late today to clena it. Indeed I did it a bit in a rush. To my to-clean-list.